frontend/src/components/nav/Nav.module.css

```diff
@@ -0,0 +1,26 @@
+.nav {
+    justify-content: space-between;
+    align-items: center;
+    margin: auto;
+}
+
+.title {
+    font-size: 2rem;
+    font-weight: 600;
+    color: var(--app-secondary-color);
+}
+
+.navList {
+    display: flex;
+}
+
+.navList > * {
+    margin: 0 0.3em;
```

Comment:
You should use consistent units - either em or rem everywhere. I'd recommend using rem everywhere.